speechbrain/nnet/ligru/cuda_ligru_cell.py

```diff
@@ -0,0 +1,223 @@
+"""This library implements a LiGRU cell needed for the LiGRU.
+
+LiGRU is a Gated Recurrent Unit introduced by Assoc. Prof Mirco 
+Ravanelli in 2018 (see: https://arxiv.org/pdf/1803.10225.pdf).
```

Comment:
Please add some infos on what must be installed on the header of this file as well.